HappyCamps/src/app/services/user.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { RegiserUserModel } from '../Models/register-user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  base_url = "https://localhost:7027/api"
+
+  constructor(private http:HttpClient) {}
+
+  registerUser(newUser:RegiserUserModel){
+    return this.http.post<any>(`${this.base_url}/User/register`,newUser);
```

Comment:
Why is the return type any?

Reply:
Because it might return an error and we can't define an error type:)